packages/functional-tests/tests/oauth/signin.spec.ts

```diff
@@ -1,12 +1,21 @@
 import { test, expect } from '../../lib/fixtures/standard';
 
+let email = '';
 const PASSWORD = 'passwordzxcv';
 
 test.describe('OAuth signin', () => {
   test.beforeEach(({}, testInfo) => {
     test.slow(testInfo.project.name !== 'local', 'email delivery can be slow');
   });
 
+  test.afterEach(async ({ target }) => {
+    try {
```

Comment:
Is there a reason the try catch is needed? It seems like this should be reliable?

Reply:
Gonna remove this, it was to cleanup account that were via `auth.signUp`